DigitalLearningSolutions.Data/Models/TrackingSystem/DateInformation.cs

```diff
@@ -5,22 +5,61 @@
 

```

Comment:
I don't really understand the whole of this file, it seems to be doing 2 things. We have the DateInformation entity, but also static methods that seem to have nothing to do with it.

Reply:
The file contains the DateInformation class and some methods to generate a date or date range label from it. One of the date range label methods doesn't actually require information from the instance of the class (it's for the edge case where there's only one data point, with a date range specified by the start and end dates from the filters) but is still in this class as it is a special case of the other two methods, used in the same place in the code.